packages/js-dapi-client/lib/BlockHeadersProvider/BlockHeadersProvider.js

```diff
@@ -19,17 +20,23 @@ const defaultOptions = {
   autoStart: false,
 };
 
-class BlockHeadersProvider {
+const EVENTS = {
+  ERROR: 'ERROR',
```

Comment:
I think we should use `error` name of the event to enable default error handling logic https://nodejs.org/api/events.html#error-events

Reply:
Done